cmd/kots/cli/install.go

```diff
@@ -614,3 +644,66 @@ func getHttpProxyEnv(v *viper.Viper) map[string]string {
 	return env
 
 }
+
+func isAppReady(namespace string, authSlug string, localPort int) string {
+
+	url := fmt.Sprintf("http://localhost:%d/api/v1/apps", localPort)
+	apps, err := getApps(url, authSlug)
+	if err != nil {
+		return ""
+	}
+	for _, app := range apps.Apps {
```

Comment:
There's a TODO here to support multiple apps.  But this is OK for now

Reply:
Let me add the TODO